### DISCUSSION TITLE
KPN ITV not working, stuck on 85%

### DISCUSSION THREAD

**Discussion by thomasVink89** [fabianishere/udm-iptv#200] [Q&A]
Hello,

I cannot get KPN ITV working with the following setup:

UDMpro
UDM Switch
KPN Arris

Internet worked easily with your setup guide. I then created a seperate network for itv, with vlan id 4 and a ip range only 192.168.4.x. Also IGMP snooping is activated. This network is then assigned to the correct port.

KNP arris receives a correct IP, but when loading it does not get past the 85% mark.

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
=== Service Logs ===
Nov 07 23:17:00 ubnt udm-iptv[7833]: udhcpc (v1.22.1) started
Nov 07 23:17:00 ubnt udm-iptv[7833]: Sending discover...
Nov 07 23:17:01 ubnt udm-iptv[7833]: Sending select for 10.129.253.154...
Nov 07 23:17:01 ubnt udm-iptv[7833]: Lease of 10.129.253.154 obtained, lease time 3600
Nov 07 23:17:01 ubnt udm-iptv[7833]: NATing IPTV network ranges (if necessary)
Nov 07 23:17:01 ubnt udm-iptv[7833]: Setting up igmpproxy
Nov 07 23:17:01 ubnt udm-iptv[7833]: Starting igmpproxy
Nov 07 23:17:01 ubnt igmpproxy[7833]: MRT_INIT failed; Errno(92): Protocol not available
Nov 07 23:17:01 ubnt systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Nov 07 23:17:01 ubnt systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

I thought the LAN interfaces was not correct, but trying to set it using the following

dpkg-reconfigure -p medium udm-iptv IPTV_LAN_INTERFACES br4

returns this

LAN_INTERFACES not installed

Please help, for my neighbour now actually having gone a step back is not to happy with my suggestions (buy a UDM pro!) and help so far.

**Comment by fabianishere** [comment]
The script is complaining that multicast routing is not available on the kernel running on your UDM Pro.

What firmware version are you using? You need at least v1.11 to use this script (or run a custom kernel).

**Reply by thomasVink89** [reply]
Hi Fabian,

The UDM is new out of the box installed yesterday. The firmware of the UDM is version 1.12.30. 

Also tried to connect the arris straight onto the UDM, instead of going via the switch. Did not work either.

The errorcode shown on screen is 

https://www.kpn.com/service/televisie/problemen/foutcodes/foutcode-563.htm?icid=foutcodetv-563

**Reply by fabianishere** [reply]
Can you provide the output of the following command:

```
zcat /proc/config.gz | grep "MROUTE"
```

The issue is related to the software running on your UDM, so for now it does not matter how you connect the Arris.
